kingpin/actors/group.py

```diff
@@ -62,6 +62,14 @@ def __init__(self, *args, **kwargs):
         # Pre-initialize all of our actions!
         self._actions = self._build_actions()
 
+    def get_orgchart(self, parent=''):
+        ret = super(BaseGroupActor, self).get_orgchart(parent=parent)
```

Comment:
What exactly is happening here? Is this a hack so that you can get a single "root orgchart object"?